match/lib/match/utils.rb

```diff
@@ -45,6 +45,55 @@ def self.environment_variable_name_profile_name(app_identifier: nil, type: nil)
       (base_environment_variable_name(app_identifier: app_identifier, type: type) + ["profile-name"]).join("_")
     end
 
+    # Fill identity(name) of certificate to env variable
+    def self.fill_environment_cert(params, identity)
+      # instead we specify the UUID of the profiles
+      key = environment_variable_name_cert(params)
+      UI.important "Setting environment variable '#{key}' to '#{identity}'" if $verbose
+      ENV[key] = identity
+    end
+
+    def self.get_cert_info(cer_certificate_path)
+      command = "openssl x509 -inform der -in #{cer_certificate_path.shellescape} -subject -dates -noout"
+      command << " &" # start in separate process
+
+      output = Helper.backticks(command, print: $verbose).split("\n")
+
+      # openssl output:
+
+      # subject= /UID={User ID}/CN={Certificate Name}/OU={Certificate User}/O={Organisation}/C={Country}\n
+      # notBefore={Start datetime}\n
+      # notAfter={End datetime}
+
+      # removing subject= in first line
+      subject_value = output[0][output[0].index('=') + 2, output[0].length]
```

Comment:
Do `.gsub(/\s*subject=\s*/, "")` and `.gsub("/", "\n")` before running `.split("\n")` instead, that's a bit more clear and doesn't introduce 2 temporaries.